specs/eventing/data-plane.md

```diff
@@ -1,125 +1,183 @@
+has handled the event (processed the event or stored it in stable storage). The
+following response codes are explicitly defined; event recipients MAY also
+respond with other response codes. A response code not in this table SHOULD be
+treated as a retriable error.
+
+| Response code | Meaning                                           | Retry | Delivery completed | Error |
+| ------------- | ------------------------------------------------- | ----- | ------------------ | ----- |
+| `1xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `200`         | [Accepted, event in reply](#derived-reply-events) | No    | Yes                | No    |
+| `202`         | Event accepted                                    | No    | Yes                | No    |
+| other `2xx`   | (Unspecified)                                     | No    | Yes                | No    |
+| `3xx`         | (Unspecified)                                     | No\*  | No\*               | Yes\* |
+| `400`         | Unparsable event                                  | No    | No                 | Yes   |
+| `404`         | Endpoint does not exist                           | Yes   | No                 | Yes   |
+| `409`         | Conflict / Processing in progress                 | Yes   | No                 | Yes   |
+| `429`         | Too Many Requests / Overloaded                    | Yes   | No                 | Yes   |
+| other `4xx`   | Error                                             | No    | No                 | Yes   |
+| `5xx`         | Error                                             | Yes   | No                 | Yes   |
+
+\* Unspecified `1xx`, `2xx`, and `3xx` response codes are **reserved for future
+extension**. Event recipients SHOULD NOT send these response codes in this spec
+version, but event senders MUST handle these response codes as errors or success
+as appropriate and implement described success or failure behavior.
+
+Recipients MUST be able to handle duplicate delivery of events (for example, via
+idempotency or tracking event delivery state) and MUST accept delivery of
+duplicate events, as the event acknowledgement could have been lost in return to
+the sender. As specified in the
+[CloudEvents specification](https://github.com/cloudevents/spec/blob/v1.0.1/primer.md#id),
+event recipients MUST use the
+[`source` and `id` attributes](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#required-attributes)
+to determine duplicate events if needed. This specification does not describe
+state requirements for recipients which need to detect duplicate events. (In
+general, senders MAY add or update other CloudEvent attributes on each delivery
+attempt; see [observability](#observability) for an example case).
+
+Where possible, event senders SHOULD re-attempt delivery of events where the
+HTTP request failed or returned a retryable status code. It is RECOMMENDED that
```

Comment:
Can we drop the "where the HTTP Request failed" part and just stick with "retryable status code"? Including the HTTP part could be interpreted as overriding the HTTP table above and imply that senders should retry on ALL non-2xx return codes.